recipes-app/iot2050-firmware-update/files/iot2050-firmware-update

```diff
@@ -9,6 +9,30 @@
 # This file is subject to the terms and conditions of the MIT License.  See
 # COPYING.MIT file in the top-level directory.
 #
+# To use this tool, a <firmware-update-package>.tar.xz and its sha256 checksum file
+# <firmware-update-package>.tar.xz.sha256 are needed.
+#
+# The <firmware-update-package>.tar.xz contains below files:
+#   - firmware.bin: The firmware to update.
+#   - firmware.bin.sha256: The sha256 checksum of the firmware.
+#   - update.conf.json: The update criteria. Currently only the OS image version
+#       is checked.
+#
+# Example of update.conf.json:
+# {
+#     "update_criteria":[
+#         {
+#             "image_type": "Example Image",
+#             "version_field": "BUILD_ID",
+#             "version_value": "V01.01.00"
+#         },
+#         {
+#             "image_type": "Industrial OS",
+#             "version_field": "VERSION_ID",
+#             "version_value": "2.1.1"
+#         }
+#     ]
+# }
```

Comment:
Hardware properties are missing here.

Reply:
Which one?

Reply:
Match PG1/2. See above. This is mandatory, or we cannot ship any update for PG1 via this updater tool if that update also covers PG2.